src/handlers/app_listview.rs

```diff
@@ -0,0 +1,45 @@
+use crate::ui::AppWindow; // Import the re-exported AppWindow type
+// use crate::ui_events::UIEvent; // Import the UIEvent enum
+use slint::Model;
+use slint::ModelRc;
+use slint::SharedString;
+use slint::VecModel;
+use slint::ComponentHandle;
+use std::rc::Rc;
+
+pub enum AppListviewEvent {
+    AddItem(SharedString),
+    // Add more events here as needed
+}
+
+pub fn init(ui: &AppWindow) {
+    let items_model: Rc<VecModel<SharedString>> =
+        Rc::new(VecModel::from(vec!["Hello".into(), "World".into()]));
+    let items_model_rc = ModelRc::from(items_model.clone());
+    ui.set_items(items_model_rc);
+    let ui_weak = ui.as_weak();
+    ui.on_request_add_item(move || {
```

Comment:
... Over here we would register them on init, using the snake_case format you get from the Slint markdown itself...